### PR TITLE
[P1-L12] Address a number of erroneous comments through the DVM

### PR DESCRIPTION
**Pull request by chrismaree**
This PR fixes a number of errors in the contracts comments.

### CODE REVIEW

File: core/contracts/oracle/implementation/ResultComputation.sol [hunk 1]
```diff
@@ -52,7 +52,7 @@ library ResultComputation {
      * @notice Returns whether the result is resolved, and if so, what value it resolved to.
      * @dev `price` should be ignored if `isResolved` is false.
      * @param data contains information against which the `minVoteThreshold` is applied.
-     * @param minVoteThreshold min number of tokens that must have been voted for the result to be valid. Can be
+     * @param minVoteThreshold min exclusive number of tokens that must have been voted for the result to be valid. Can be
```
Comment (mrice32) [inline core/contracts/oracle/implementation/ResultComputation.sol:55]:
```suggestion
     * @param minVoteThreshold min (exclusive) number of tokens that must have been voted for the result to be valid. Can be
```

File: core/contracts/oracle/implementation/Voting.sol [hunk 2]
```diff
@@ -546,7 +546,7 @@ contract Voting is Testable, Ownable, OracleInterface, VotingInterface, Encrypte
         uint currentRoundId = voteTiming.computeCurrentRoundId(blockTime);
 
         // Solidity memory arrays aren't resizable (and reading storage is expensive). Hence this hackery to filter
-        // `pendingPriceRequests` only to those requests that `isActive()`.
+        // `pendingPriceRequests` only to those requests that has an Active RequestStatus.
```
Comment (mrice32) [inline core/contracts/oracle/implementation/Voting.sol:549]:
```suggestion
        // `pendingPriceRequests` only to those requests that have an Active RequestStatus.
```

File: core/contracts/oracle/implementation/Governor.sol [hunk 3]
```diff
@@ -139,7 +138,7 @@ contract Governor is MultiRole, Testable {
         emit ProposalExecuted(id, transactionIndex);
     }
 
-    /***************************************
+    /****************************************
```
Comment (nicholaspai) [inline core/contracts/oracle/implementation/Governor.sol:141]: YUGE

File: core/contracts/oracle/implementation/DesignatedVoting.sol [hunk 4]
```diff
@@ -19,7 +19,7 @@ contract DesignatedVoting is Withdrawable {
      ****************************************/
 
     enum Roles {
-        Owner, // Can set the Voter and Withdrawer roles.
+        Owner, // Can set the Voter role. Is also permissioned to be the Withdrawer from initialization.
```
Comment (nicholaspai) [inline core/contracts/oracle/implementation/DesignatedVoting.sol:22]: I would make this more specific and say that the `Owner` and the `Withdrawer` are permanently the same role.

File: core/contracts/oracle/implementation/Registry.sol [hunk 5]
```diff
@@ -73,7 +73,7 @@ contract Registry is RegistryInterface, MultiRole {
      * @notice Registers a new financial contract.
      * @dev Only authorized contract creators can call this method.
      * @param parties an array of addresses who become parties in the contract.
-     * @param contractAddress defines the address of the deployed finan.
+     * @param contractAddress address of the contract against which the parties are registered.
```
Comment (nicholaspai) [inline core/contracts/oracle/implementation/Registry.sol:76]: add a `with` between `against` and `which`

File: core/contracts/oracle/implementation/ResultComputation.sol [hunk 6]
```diff
@@ -52,7 +52,7 @@ library ResultComputation {
      * @notice Returns whether the result is resolved, and if so, what value it resolved to.
      * @dev `price` should be ignored if `isResolved` is false.
      * @param data contains information against which the `minVoteThreshold` is applied.
-     * @param minVoteThreshold min number of tokens that must have voted for the result to be valid. Can be
+     * @param minVoteThreshold min (exclusive) number of tokens that must have been voted for the result to be valid. Can be
```
Comment (mrice32) [inline core/contracts/oracle/implementation/ResultComputation.sol:55]:
```suggestion
     * @param minVoteThreshold min (exclusive) number of tokens that must have voted for the result to be valid. Can be
```

**Pull request merged**